packages/sources/css/src/components/radio-button/src/index.css

```diff
@@ -0,0 +1,89 @@
+/* Design tokens */
+@import '@vtmn/css-design-tokens/src/colors';
+@import '@vtmn/css-design-tokens/src/spacings';
+@import '@vtmn/css-design-tokens/src/typography';
+
+body {
+    font-family: var(--vtmn-typo_font-family);
+  }
+  
+  p {
+    margin: 8px 0;
+  }
+  
+  [type="radio"] {
```

Comment:
It's better to use a specific css class here like ".vtmn-radio-button[type="radio"]"